src/app/routes/onDemandRadio/getInitialData/logInitialData/index.test.js

```diff
@@ -0,0 +1,43 @@
+import path from 'ramda/src/path';
+import { parsePageIdentifier } from '.';
+import onDemandRadioEpisodeJson from '#data/pashto/bbc_pashto_radio/w3ct0lz1';
+import onDemandRadioBrandJson from '#data/indonesia/bbc_indonesian_radio/w13xtt0s';
+
+describe('Logging get initial data', () => {
```

Comment:
Do we want to maybe add a test in here (or perhaps in getInitialData) which checks that the radio_expired_episode actually gets logged?

Reply:
Added a positive and negative test scenario for the logger being called. Let me know what you think!